bcdi/simulation/supportMaker.py

```diff
@@ -0,0 +1,414 @@
+# -*- coding: utf-8 -*-
+
+# BCDI: tools for pre(post)-processing Bragg coherent X-ray diffraction imaging data
+#   (c) 07/2017-06/2019 : CNRS UMR 7344 IM2NP
+#   (c) 07/2019-present : DESY PHOTON SCIENCE
+#       authors:
+#         Steven Leake, steven.leake@esrf.fr
+#         makePoly commands adapted from Ross Harder scripts decades old
+"""Make a 3D polygon."""
+
+import numpy as np
+import h5py as h5
+import sys
+
+helptext = """
+The concept is to be able to build a support from a set of defined planes these
+planes can be positioned based on physical size (nm) if known or eventually made
+into a fancy tool with a 3D view of the support versus the data so you can match
+fringes. It may also be interesting to consider how to make 3D supports from
+other characterisation methods (SEM etc...).
+"""
```

Comment:
```suggestion
"""
The concept is to be able to build a support from a set of defined planes these
planes can be positioned based on physical size (nm) if known or eventually made
into a fancy tool with a 3D view of the support versus the data so you can match
fringes. It may also be interesting to consider how to make 3D supports from
other characterisation methods (SEM etc...).
"""
```